tensorflow_blade/build.py

```diff
@@ -251,10 +255,22 @@ def _write(line, cmd="build"):
             _write("--//:device=cpu")
```

Comment:
`aarch64` seems to be different from `cpu`, `cpu` here means x86 i think.

Reply:
I think these names should be unified for disc/tf_blade/torch_blade，maybe discuss offline?